src/index.ts

```diff
@@ -73,31 +120,84 @@ modeButtons.forEach(button => {
 });
 
 /**
- * If there is no current mode creates the listeners. Hides non cutTools.
- * Sets the current mode to cut mode.
+ * Sets the current mode to cut mode. Hides non cutTools.f
  */
 function cutMode() {
-    modeState = "cutMode";
+    modeState = Mode.cutMode;
     cutTools.style.display = "block";
     //Block all other mode tools
     atomTools.style.display = "none";
 }
 
 /**
- * Sets the current mode to atom mode. Hides non atomTools.
+ * Sets the current mode to atom mode. Hides non atom tools.
  */
 function atomMode() {
-    modeState = "atomMode";
+    modeState = Mode.atomMode;
     atomTools.style.display = "block";
     //Block all other mode tools
     cutTools.style.display = "none";
 }
 
 /**
- * Sets the current mode to move mode. Hides non moveTools.
+ * Sets the current mode to move mode. Hides non move tools.
  */
 function dragMode() {
-    modeState = "dragMode";
+    modeState = Mode.dragMode;
+    cutTools.style.display = "none";
+    atomTools.style.display = "none";
+}
+
+/**
+ * Sets the current mode to moveSingleMode. Hides non move tools.
+ */
+function moveSingleMode() {
+    modeState = Mode.moveSingleMode;
+    cutTools.style.display = "none";
+    atomTools.style.display = "none";
+}
+
+/**
+ * Sets the current mode to moveMultiMode. Hides non move tools.
+ */
+function moveMultiMode() {
+    modeState = Mode.moveMultiMode;
+    cutTools.style.display = "none";
+    atomTools.style.display = "none";
+}
+
+/**
+ * Sets the current mode to copySingleMode. Hides non copy tools.
+ */
+function copySingleMode() {
+    modeState = Mode.copySingleMode;
+    cutTools.style.display = "none";
+    atomTools.style.display = "none";
+}
+
+/**
+ * Sets the current mode to copyMultiMode. Hides non copy tools.
+ */
+function copyMultiMode() {
+    modeState = Mode.copyMultiMode;
+    cutTools.style.display = "none";
+    atomTools.style.display = "none";
+}
+
+/**
+ * Sets the current mode to deleteSingleMode. Hides non delete tools.
+ */
+function deleteSingleMode() {
+    modeState = Mode.deleteSingleMode;
+    cutTools.style.display = "none";
+    atomTools.style.display = "none";
+}
+
+/**
+ * Sets the current mode to deleteMultiMode. Hides non delete tools.
+ */
+function deleteMultiMode() {
+    modeState = Mode.deleteMultiMode;
     cutTools.style.display = "none";
     atomTools.style.display = "none";
 }
```

Comment:
Too much code reuse. Combine all of these into a single function `setMode` that takes a mode Enum for the mode. Export the Mode Enum, and from the HTML button onClicks call `setMode(Mode.somethingMode)` for each mode button.